src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -161,6 +172,14 @@ Copyright (C) Microsoft Corporation. All rights reserved.
     <ProduceReferenceAssembly Condition="'$(ProduceReferenceAssembly)' == ''">false</ProduceReferenceAssembly>
   </PropertyGroup>
 
+  <PropertyGroup Condition=" $(OutputPath) == '' ">
+    <OutputPath Condition=" '$(PlatformName)' == 'AnyCPU' ">$(BaseOutputPath)$(Configuration)\</OutputPath>
+    <OutputPath Condition=" '$(PlatformName)' != 'AnyCPU' ">$(BaseOutputPath)$(PlatformName)\$(Configuration)\</OutputPath>
+  </PropertyGroup>
+  <PropertyGroup>
+    <OutputPath>$([MSBuild]::EnsureTrailingSlash('$(OutputPath)'))</OutputPath>
+  </PropertyGroup>
+
   <PropertyGroup Condition=" '$(OutputPath)' == '' ">
     <!--
         A blank OutputPath at this point means that the user passed in an invalid Configuration/Platform
```

Comment:
If we're allowing defaults for `OutputPath`, then, how can we check for invalid configuration/platform combinations?

1. We could check for `PlatformTarget` instead of `OutputPath` but that won't be a complete solution.
2. We could send in the available combinations, and check against current value but that'll only work with solution that defines all combinations.
3. There are no invalid configurations/platform (only missing properties). In this case, let the project system handle it.

Any suggestions?

Reply:
Similar to `OutDir` and `OutDirWasSpecified`...?

We can do that. I'll try it and see how it goes!

Eventually, We should ignore `OutputPath` as a means of checking invalid Configuration/Platforms.